packages/blocks/src/api/raw-handling/index.js

```diff
@@ -225,3 +229,41 @@ export default function rawHandler( { HTML = '', plainText = '', mode = 'AUTO',
 
 	return blocks;
 }
+
+/**
+ * Converts an HTML string to known blocks.
+ *
+ * @param {string} $1.HTML The HTML to convert.
+ *
+ * @return {Array} A list of blocks.
+ */
+export function rawHandler( { HTML = '' } ) {
+	// If we detect block delimiters, parse entirely as blocks.
+	if ( HTML.indexOf( '<!-- wp:' ) !== -1 ) {
+		return parseWithGrammar( HTML );
+	}
+
+	// An array of HTML strings and block objects. The blocks replace matched
+	// shortcodes.
+	const pieces = shortcodeConverter( HTML );
+	const rawTransformations = getRawTransformations();
+	const blockContentSchema = getBlockContentSchema( rawTransformations );
+
+	return compact( flatMap( pieces, ( piece ) => {
+		// Already a block from shortcode.
+		if ( typeof piece !== 'string' ) {
+			return piece;
+		}
+
+		const filters = [
```

Comment:
How do we decide which filters are to be included here when one of the main points of distinguishing raw / paste is the strictness of its filtering?

Reply:
I'll add a comment. These are the only filters that are really needed for us to be able to transform the HTML into blocks. One is for the read more tag, one is to ensure images etc are properly positioned, and the last is to ensure text content in quotes without a wrapper tag are wrapped in a paragraph. None of these remove any content.